doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -280,7 +275,7 @@ Requests to the environment
    .. cmdv:: Search { + [-]@term_pattern_string }
 
       where
-      :n:`@term_pattern_string` is a term_pattern, a string, or a string followed
+      :production:`term_pattern_string` is a term_pattern, a string, or a string followed
```

Comment:
I don't like this verbal definition, I should add a new `productionlist` instead. There is also the question of the definition of `term_pattern` and how it related to `cpattern` that is described in the Ltac chapter, and to the grammar of Coq terms, and the section on existential variables of the Gallina extensions chapter.